### PR TITLE
Add Support  : pixel 7/ 7a/ 7p

### PR DESCRIPTION
**Pull request by dominator-prime**

### CODE REVIEW

File: build_targets [hunk 1]
```diff
@@ -2,15 +2,18 @@
 # <device> <build_type> <auto-upload> <upload-recovery>
 beryllium userdebug yes yes
 chef userdebug yes yes
+cheetah userdebug yes yes
```
Comment (gwolf2u) [inline build_targets:5]: missing cheetah repo?
android_device_google_cheetah else won't find it automatically

File: build_targets [hunk 2]
```diff
@@ -2,15 +2,18 @@
...
 ginkgo userdebug yes yes
 channel userdebug yes yes
 guacamole userdebug yes yes
 hotdogg userdebug yes yes
+lynx userdebug yes yes
```
Comment (gwolf2u) [inline build_targets:10]: set dependencies file

File: build_targets [hunk 3]
```diff
@@ -2,15 +2,18 @@
...
 lemonade userdebug yes yes
 lemonadep userdebug yes yes
 lemonades userdebug yes yes
 mars userdebug yes yes
 miatoll userdebug yes yes
+pantah userdebug yes yes
```
Comment (gwolf2u) [inline build_targets:16]: set dependencies file

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: build_targets (hunk 1 above)
Comment (dominator-prime) [reply build_targets:5]: actually panther  ,  cheetah  using unified dt

Comment (gwolf2u) [reply build_targets:5]: well won't work that way
on init it'll search on crDroid org for said repo in specific format
if not found, will bail out
https://github.com/crdroidandroid/android_vendor_crdroid/commit/11e962b27f9ce407ed12c3532b42a3080eac3a81

**Comment by dominator-prime** [comment]
let me create 1 by 1 for  7 , 7a , 7p

**Pull request closed without merging**